lib/inspec/plugin/v2/gem_source_manager.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+require "singleton" unless defined?(Singleton)
+require "forwardable" unless defined?(Forwardable)
+require "chef-licensing"
+
+module Inspec::Plugin::V2
+  class GemSourceManager
+    include Singleton
+    extend Forwardable
+
+    DEFAULT_CHEF_RUBY_GEMS_SERVER = "rubygems.chef.io"
+    DEFAULT_USERNAME = "v1"
```

Comment:
Not a major concern, but is `DEFAULT_USERNAME` the right name to use in this context?

Reply:
it is because that is the default username and in future this could become v2 may be but it is out of scope currently